web-ui/src/app/pages/multi-step-page/step-buttons.component.ts

```diff
@@ -1,16 +1,15 @@
 import { Component, EventEmitter, Input, Output } from '@angular/core';
-import { GlobalState, Step } from './steps';
 
 @Component({
     selector: 'grt-step-buttons',
     templateUrl: './step-buttons.component.html'
 })
 export class StepButtonsComponent {
     @Input()
-    public currentStep: Step<GlobalState>;
+    public currentStep: number;
 
     @Input()
-    public steps: Step<GlobalState>[];
+    public steps: number;
 
     @Output()
     public prev = new EventEmitter();
```

Comment:
Waar worden deze ge-emit?

Reply:
In de `ExampleBasedSearchComponent` en `XpathSearchComponent`.